docker-compose.yml

```diff
@@ -22,6 +22,8 @@ services:
       - POSTGRES_USER=art-peace-user
       - POSTGRES_DB=art-peace-db
   backend:
+    volumes:
+    - ./configs:/app/configs
```

Comment:
The configs are stored at `/configs` from docker build. Also the dockerfile is being used to replace `docker-backend.config.json` with `backend.config.json`. Using volumes wont be able to carry out that change, so the path will need to be changed in the command line arguments of the backend startup script.
It could be done, but not sure if its worth it since you'll still need to restart the backend to reload the config, also the backend config barely ever changes so might not be worth creating a volume for it.